lib/elixir/test/elixir/task_test.exs

```diff
@@ -177,64 +173,70 @@ defmodule TaskTest do
     end
 
     test "exits on normal exit" do
-      task = Task.async(fn -> exit :normal end)
+      task = Task.async(fn -> exit(:normal) end)
       assert catch_exit(Task.await(task)) == {:normal, {Task, :await, [task, 5000]}}
     end
 
     test "exits on task throw" do
       Process.flag(:trap_exit, true)
-      task = Task.async(fn -> throw :unknown end)
+      task = Task.async(fn -> throw(:unknown) end)
+
       assert {{{:nocatch, :unknown}, _}, {Task, :await, [^task, 5000]}} =
-             catch_exit(Task.await(task))
+               catch_exit(Task.await(task))
     end
 
     test "exits on task error" do
       Process.flag(:trap_exit, true)
       task = Task.async(fn -> raise "oops" end)
-      assert {{%RuntimeError{}, _}, {Task, :await, [^task, 5000]}} =
-             catch_exit(Task.await(task))
+      assert {{%RuntimeError{}, _}, {Task, :await, [^task, 5000]}} = catch_exit(Task.await(task))
     end
 
     test "exits on task undef module error" do
       Process.flag(:trap_exit, true)
       task = Task.async(&:module_does_not_exist.undef/0)
-      assert {{:undef, [{:module_does_not_exist, :undef, _, _} | _]},
-              {Task, :await, [^task, 5000]}} =
-             catch_exit(Task.await(task))
+
+      assert {
+               {:undef, [{:module_does_not_exist, :undef, _, _} | _]},
+               {Task, :await, [^task, 5000]}
+             } = catch_exit(Task.await(task))
```

Comment:
Please rewrite this assertion so that you first match on the 2-elem tuple and then match deeper

```elixir
assert {exit, mfa} = catch...
assert {:undef, ...} = exit
...
```

so that the formatting is nicer.